test/test_transform.js

```diff
@@ -253,7 +253,7 @@ describe('build plots', () => {
                  `Wrong X axis`)
     assert.equal(plot.layer[0].encoding.y.field, 'green',
                  `Wrong Y axis`)
-    assert(!('color' in plot.layer[0].encoding),
+    assert(!(' ' in plot.layer[0].encoding),
```

Comment:
hm - this is going to be hard for the next reader to figure out...

Reply:
does `assert(plot.layer[0].encoding.color.field === null, `Should not have color`)` resolve this/make it more human read-able? [see latest commits]

Reply:
You can use `assert.strictEqual(something, null, message)` to get an "is exactly null with no weird type conversion" check.